jetbrains-core/src/software/aws/toolkits/jetbrains/services/cloudwatch/logs/CloudWatchLogsServiceNode.kt

```diff
@@ -0,0 +1,45 @@
+// Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package software.aws.toolkits.jetbrains.services.cloudwatch.logs
+
+import com.intellij.openapi.project.Project
+import icons.AwsIcons
+import software.amazon.awssdk.services.cloudwatchlogs.CloudWatchLogsClient
+import software.amazon.awssdk.services.cloudwatchlogs.model.LogGroup
+import software.aws.toolkits.jetbrains.core.explorer.nodes.AwsExplorerNode
+import software.aws.toolkits.jetbrains.core.explorer.nodes.AwsExplorerResourceNode
+import software.aws.toolkits.jetbrains.core.explorer.nodes.AwsExplorerServiceNode
+import software.aws.toolkits.jetbrains.core.explorer.nodes.CacheBackedAwsExplorerServiceRootNode
+import software.aws.toolkits.jetbrains.services.cloudwatch.logs.resources.CloudWatchResources
+
+class CloudWatchLogsServiceNode(project: Project, service: AwsExplorerServiceNode) : CacheBackedAwsExplorerServiceRootNode<LogGroup>(
+    project,
+    service,
+    CloudWatchResources.LIST_LOG_GROUPS
+) {
+    override fun toNode(child: LogGroup): AwsExplorerNode<*> = CloudWatchLogsNode(nodeProject, child.arn(), child.logGroupName())
+}
+
+class CloudWatchLogsNode(
+    project: Project,
+    val arn: String,
+    private val logGroupName: String
+) : AwsExplorerResourceNode<String>(
+    project,
+    CloudWatchLogsClient.SERVICE_NAME,
+    logGroupName,
+    // TODO get icon for CloudWatch
+    AwsIcons.Resources.CLOUDFORMATION_STACK
+) {
+    override fun resourceType() = "group"
```

Comment:
I almost forgot about the magic action group construction system 😢

Reply:
I did forget and was wondering why it wasn't working